src/disasm.cpp

```diff
@@ -444,8 +445,11 @@ void jl_dump_asm_internal(uintptr_t Fptr, size_t Fsize, int64_t slide,
                   TripleName.c_str());
         return;
     }
-
     unsigned OutputAsmVariant = 0; // ATT or Intel-style assembly
+
+    if( strcmp(asm_variant, "intel")==0){
```

Comment:
preferred style would be `if (strcmp(asm_variant, "intel")==0) {` or `if (strcmp(asm_variant, "intel") == 0) {`